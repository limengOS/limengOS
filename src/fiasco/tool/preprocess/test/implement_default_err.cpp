INTERFACE:

class Test
{
public:
  void err_func1();
};

IMPLEMENTATION:

IMPLEMENT_OVERRIDE void Test::err_func1() { /* this is the override */ }
