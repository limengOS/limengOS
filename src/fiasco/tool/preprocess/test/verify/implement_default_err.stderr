implement_default_err.cpp:10: error: No default implementation for Test::err_func1 found
